Framework/PythonInterface/test/python/plugins/algorithms/FindSXPeaksConvolveTest.py

```diff
@@ -42,9 +42,9 @@ def setUpClass(cls):
         axis = cls.ws.getAxis(0)
         axis.setUnit("TOF")
         # fake peak centred on ispec=30 (detid=79) and TOF=5 - near middle of bank
-        peak_1D = array([0, 0, 0, 0, 4, 6, 4, 0, 0, 0, 0, 0, 0])
-        cls.ws.setY(30, cls.ws.readY(30) + peak_1D)
```

Comment:
Do you know why this spectrum (previously corresponding to the middle of the bank) needed changing - I thought it would only be the second peak on the edge not found?

Reply:
It wasn't finding any peaks with the previous settings. I think I should have just increased the intensity without changing the spectrum. I will revert the spectrum number change and see if it is working.

Reply:
It might be that the peak is too near the edge even in the middle of the bank - we could increase the bank size if your approach doesn't work?

Reply:
Ok, I will try that

Reply:
It failed again after increasing the number of pixels from 7 to 14